model/profiles/eth_nic_settings.yaml

```diff
@@ -0,0 +1,81 @@
+components:
+  schemas:
+    Profiles.EthNicSetting:
```

Comment:
looks good, just one observation, we need to make a better hierarchy, like profiles.dataflow.host_management.ethnicsettings

Reply:
isn t the point of profiles to keep them in a general area and allow their usage anywhere else? i know that right now each profile has only a specific region where it should be, but i m thinking that it s better to leave this flexibility in the future, to be able to add any profile anywhere if needed.